daffodil-core/src/main/scala/org/apache/daffodil/dsom/ElementBase.scala

```diff
@@ -406,7 +411,7 @@ trait ElementBase
    * To be properly constructed, scala's xml Elems must share the scope (namespace bindings) of the enclosing
    * parent element, except when it adds more of its own bindings, in which case the tail is supposed to be shared.
    */
-  private lazy val minimizedScope: NamespaceBinding = {
+  private lazy val minimizedScope: NamespaceBinding = LV('minimizedScope) {
```

Comment:
Minimized scope isn't correct any more because a shared definition may be reused in multiple contexts that have different enclosing sets of namespace prefix definitions. 

Really we need a table here, so we can lookup the distinct minimizedScope for that context.

Reply:
This is not true yet. This will be true once we start sharing all the components.